### PR TITLE
new test for TSWEB-132

### PR DESCRIPTION
**Pull request by YevheniiKel**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 1]
```diff
@@ -18,6 +18,33 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+
+	await baseDriverSteps.checkUrl(`${UrlProvider.careerUrl()}uk-UA`);
+	await expect((await containerSteps.getContainer(ContainerByClass, Containers.standWithUkraineTitleClass)).Element).toHaveText(
+		'Відповідь Techstack на війну в Україні',
+		{timeout: 1000}
+	);
+	await expect(await driver.getByTestId(Button.LearnMoreButton)).toHaveText('Ознайомитися');
+
+	await driver.getByTestId(Button.LearnMoreButton).click();
+	await baseDriverSteps.checkUrl('https://tech-stack.com/blog/techstack-stands-with-ukraine/');
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:41]: Add this URL to the UrlPath class and use UrlPath.YourURL instead

**Review by VladFedash — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/identifiers/Button.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export default class Button {
 	static Menu = 'HeaderOpenMenuButton-HeaderOpenMenuButton';
 	static ContactUs = 'Button-ContactUs';
 	static Close = 'CloseButton-Close';
+	static LearnMoreButton = 'LearnMoreButton-SharedLearnMore2'
```
Comment (VladFedash) [inline ui_tests_playwright/automation/identifiers/Button.ts:16]: Does it a problem that the button data-id has a number at the end?

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 3]
```diff
@@ -18,6 +18,33 @@ test('Check that user can switch language in navigation header @Regression @Navi
 	await expect(uaButtonSwitcher).toHaveClass(/active-locale/);
 });
 
+test('Check the the "Stand with Ukraine" block with localization @Regression @StandWithUkraine @TSWEB-132', async () => {
+	await expect((await containerSteps.getContainer(ContainerByClass, Containers.standWithUkraineClass)).Element).toBeVisible();
+	await expect((await containerSteps.getContainer(ContainerByClass, Containers.standWithUkraineTitleClass)).Element).toHaveText(
+		'Techstack stands with Ukraine'
+	);
+	await expect(await driver.getByTestId(Button.LearnMoreButton)).toHaveText('Learn More');
```
Comment (anastasiia-liashova) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:26]: without await 
await expect(driver.getByTestId....

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 4]
```diff
@@ -18,6 +18,33 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+
+	const buttonSwitcher = await (
```
Comment (anastasiia-liashova) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:28]: same

**Review by anastasiia-liashova — CHANGES_REQUESTED** [review]

**Review by VladFedash — APPROVED** [review]

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 5]
```diff
@@ -18,6 +20,33 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+	await baseDriverSteps.checkUrl(`${UrlProvider.careerUrl()}uk-UA`);
+	await expect(driver.getByTestId(Containers.standWithUkraineTitle)).toHaveText(
+		'Відповідь Techstack на війну в Україні',
+		{timeout: 1000}
+	);
+	await expect(driver.getByTestId(Button.LearnMoreButton)).toHaveText('Ознайомитися');
+
+	await driver.getByTestId(Button.LearnMoreButton).click();
+	await baseDriverSteps.checkUrl(UrlProvider.urlBuilder(UrlPath.Blog_StandWithUkraine, Environment.Production));
+	// Uncomment when the data-id is added for the title element
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:44]: Recommend to wait while jira-api logic will be pushed, and then push the test but with the appropriate ticket

**Review by DaniilVostrikov — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 6]
```diff
@@ -18,6 +21,32 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+	await expect(driver.getByTestId(Containers.StandWithUkraineTitle)).toHaveText('Techstack stands with Ukraine');
+	await expect(driver.getByTestId(Button.LearnMoreButton)).toHaveText('Learn More');
+
+	const buttonSwitcher = (await containerSteps.getContainer(ContainerByClass, Containers.NavigationHeaderClass)).Element.getByTestId(Button.UaLanguageSwitcher);
+	await buttonSwitcher.click();
+
+	await baseDriverSteps.checkUrl(`${UrlProvider.careerUrl()}uk-UA`);
+	await expect(driver.getByTestId(Containers.StandWithUkraineTitle)).toHaveText(
+		'Відповідь Techstack на війну в Україні',
+		{timeout: 1000}
```
Comment (anastasiia-liashova) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:39]: try without "timeout"

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 7]
```diff
@@ -18,6 +21,32 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+	);
+	await expect(driver.getByTestId(Button.LearnMoreButton)).toHaveText('Ознайомитися');
+
+	await driver.getByTestId(Button.LearnMoreButton).click();
+	await baseDriverSteps.checkUrl(UrlProvider.urlBuilder(UrlPath.Blog_StandWithUkraine, Environment.Production));
+	await expect((await driver.getByTestId(Blog.Blog_StandWithUkraineTitile))).toContainText(
```
Comment (anastasiia-liashova) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:45]: without second await

**Review by anastasiia-liashova — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 8]
```diff
@@ -18,6 +21,32 @@ test('Check that user can switch language in navigation header @Regression @Navi
 	await expect(uaButtonSwitcher).toHaveClass(/active-locale/);
 });
 
+test('Check the the "Stand with Ukraine" block with localization @Regression @StandWithUkraine @TSWEB-132', async () => {
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:24]: Check that*

File: ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts [hunk 9]
```diff
@@ -18,6 +21,32 @@ test('Check that user can switch language in navigation header @Regression @Navi
...
+	await expect(
+		(
+			await containerSteps.getContainer(ContainerByClass, Containers.StandWithUkraineClass)
+		).Element
+	).toBeVisible();
+	await expect(driver.getByTestId(Containers.StandWithUkraineTitle)).toHaveText('Techstack stands with Ukraine');
+	await expect(driver.getByTestId(Button.LearnMoreButton)).toHaveText('Learn More');
+
+	const buttonSwitcher = (await containerSteps.getContainer(ContainerByClass, Containers.NavigationHeaderClass)).Element.getByTestId(Button.UaLanguageSwitcher);
+	await buttonSwitcher.click();
```
Comment (VladFedash) [inline ui_tests_playwright/automation/test/ui/navigationHeader/NavigationHeaderTests.test.ts:34]: What is the reason for creating the buttonSwitcher variable? You don't use this const anywhere more.

**Review by VladFedash — CHANGES_REQUESTED** [review]

**Review by anastasiia-liashova — APPROVED** [review]

**Review by VladFedash — APPROVED** [review]

**Pull request merged**